x/launch/types/message_create_chain.go

```diff
@@ -69,5 +72,10 @@ func (msg *MsgCreateChain) ValidateBasic() error {
 		return sdkerrors.Wrapf(ErrInvalidInitialGenesis, "hash of custom genesis must be sha256")
```

Comment:
```suggestion
		return sdkerrors.Wrap(ErrInvalidInitialGenesis, "hash of custom genesis must be sha256")
```

Reply:
we can also fix the previous statement. The `Wrapf` is not needed. We can use `Wrap` only